go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/cosmos/cosmos-sdk v0.44.5
 	github.com/cosmos/go-bip39 v1.0.0
-	github.com/cosmos/iavl v0.17.2
+	github.com/cosmos/iavl v0.17.3
```

Comment:
The patch fixed an edge case in an incorrect manner, that puts another blocking mutex contention in every query. I guess we can just fix later in our later improvements, since its not a big impact until #604 anyway.